From bc3daab9e303018678c1cac5eb27f07ee0e12864 Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Mon, 26 Oct 2026 21:26:29 +0100
Subject: [PATCH] Verify source signatures with openpgpverify

Replace the GnuPG-based %gpgverify with %openpgpverify, which uses
Sequoia sqv and takes the same options, as part of the Fedora 45 Change
https://fedoraproject.org/wiki/Changes/Sequoia_openpgpverify

Refresh newsboat.pgp from keyservers/WKD: same certificate(s), with
self-signatures our copy was missing.

Assisted-by: Claude Opus 5.5
---
 newsboat.spec | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/newsboat.spec b/newsboat.spec
index c56ae04..f53ac02 100644
--- a/newsboat.spec
+++ b/newsboat.spec
@@ -35,6 +35,7 @@ BuildRequires: pkgconfig(stfl)
 # Rust parts
 BuildRequires:  git-core
 BuildRequires:  rust-packaging
+BuildRequires: openpgpverify
 
 Obsoletes: newsbeuter < 2.9-29
 
@@ -44,7 +45,7 @@ Provides: podboat = %{version}-%{release}
 Newsboat is a fork of Newsbeuter, an RSS/Atom feed reader for the text console.
 
 %prep
-%{gpgverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
+%{openpgpverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
 # Note to self: do not use -S git_am for released builds, it messes up version string
 %autosetup -p1
 %cargo_prep
-- 
2.56.0

